plugins/extra/p2js/componentConfigs/P2BodyConfig.ts

```diff
@@ -24,6 +25,7 @@ export default class P2BodyConfig extends SupCore.Data.Base.ComponentConfig {
     shape: { type: "enum", items: [ "box", "circle" ], mutable: true },
     width: { type: "number", min: 0, mutable: true },
     height: { type: "number", min: 0, mutable: true },
+    angle: { type: "number", min: -360, max: 360, mutable: true },
```

Comment:
Shouldn't we just go with -180 to 180? I guess that should be enough?
If yes, we should also fix here https://github.com/superpowers/superpowers-game/pull/140/files#diff-7066a6f42a064263978742ca1fe7f638R66